x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts

```diff
@@ -22,10 +22,82 @@ import {
   EntryMatch,
   EntryMatchAny,
   EntriesArray,
```

Comment:
Lots of changes in this file because I refactored the code a bit to keep it more DRY, especially since I was going to use similar expected values for the tests I was adding. The end result looks like this:
<img width="757" alt="Screen Shot 2020-07-12 at 9 08 45 AM" src="https://user-images.githubusercontent.com/616158/87258734-eaee7280-c473-11ea-8d15-5d5f4974cd74.png">
<img width="729" alt="Screen Shot 2020-07-12 at 9 09 11 AM" src="https://user-images.githubusercontent.com/616158/87258735-eb870900-c473-11ea-8257-2f50e29829b2.png">
<img width="927" alt="Screen Shot 2020-07-12 at 9 09 37 AM" src="https://user-images.githubusercontent.com/616158/87258736-eb870900-c473-11ea-86e1-53a87032eadc.png">